include/Options/Business/Nav_Menus.php

```diff
@@ -45,36 +45,23 @@ protected function get_default() {
 	 * @since 3.7
 	 *
 	 * @return array Partial schema.
-	 *
-	 * @phpstan-return array{
-	 *     type: 'object',
-	 *     patternProperties: array{
-	 *         '^\w+$': array{
-	 *             type: 'object',
-	 *             context: list<'edit'>,
-	 *             patternProperties: array{
-	 *                 '^[a-z_-]+$': array{
-	 *                     type: 'integer',
-	 *                     minimum: 0
-	 *                 }
-	 *             },
-	 *             additionalProperties: false
-	 *         }
-	 *     },
-	 *     additionalProperties: false
-	 * }
 	 */
 	protected function get_data_structure(): array {
 		return array(
 			'type'                 => 'object', // Correspond to associative array in PHP, @see{https://developer.wordpress.org/rest-api/extending-the-rest-api/schema/#primitive-types}.
 			'patternProperties'    => array(
 				'^\\w+$' => array( // Any word characters as key, correspond to a theme slug.
-					'type'              => 'object',
-					'context'           => array( 'edit' ),
-					'patternProperties' => array(
-						'^[a-z_-]+$' => array( // Language slug as key.
-							'type'    => 'integer',
-							'minimum' => 0, // A post ID.
+					'type'                 => 'object',
+					'patternProperties'    => array(
+						'^\\w+$' => array( // Any word characters as key, correspond to a nav menu location.
```

Comment:
Looks like it accept special char as well, isn't it?
See https://github.com/WordPress/wordpress-develop/blob/0ac1eb312aeb5898c3037e68d1186658236f76d8/src/wp-includes/nav-menu.php#L95

Reply:
I stupidly copied what was done for the theme slug, but looking at this with more attention now, I think both are wrong and we should use `[\w-]+` here for the location - see https://github.com/WordPress/WordPress/blob/6.6.1/wp-includes/rest-api/endpoints/class-wp-rest-menu-locations-controller.php#L53 - and `[^\/:<>\*\?"\|]+` for the theme slug - see https://github.com/WordPress/WordPress/blob/6.6.1/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php#L23.

Reply:
Indeed, better with what you've found! 👍